.github/workflows/ci.yml

```
@@ -17,3 +19,42 @@ jobs:
             echo "Error: The <stages> tag is missing from the Haar Cascade XML file."
             exit 1
           fi
+
+  validate-python-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)